sdk/src/HostRunner.ts

```
@@ -250,6 +257,13 @@ export class HostRunner implements HostExecutor {
    * when no host was supplied (legacy explicit-model path).
    */
   private readonly hostPolicy: HostExecutionPolicy | undefined;
+  /**
+   * Description rewrites applied after visibility filtering. Stored so
+   * `withOptions` re-runs them against the raw `Tool[]` under a new host.
+   */
+  private readonly toolDescriptionOverrides:
+    | Readonly<Record<string, string>>
+    | undefined;
```

ESLint

[error] Replace `|·Readonly<Record<string,·string>>⏎···` with `Readonly<Record<string,·string>>`

(prettier/prettier)